src/veins/modules/utility/ConstsPhy.h

```diff
@@ -26,68 +26,68 @@
 #include "veins/veins.h"
 
 /** @brief Modulation and coding scheme to be used for transmission */
-enum PHY_MCS {
+enum class MCS {
     // use the default MCS
-    MCS_DEFAULT = -1,
-    MCS_OFDM_BPSK_R_1_2,
-    MCS_OFDM_BPSK_R_3_4,
-    MCS_OFDM_QPSK_R_1_2,
-    MCS_OFDM_QPSK_R_3_4,
-    MCS_OFDM_QAM16_R_1_2,
-    MCS_OFDM_QAM16_R_3_4,
-    MCS_OFDM_QAM64_R_2_3,
-    MCS_OFDM_QAM64_R_3_4
+    undefined = -1,
+    ofdm_bpsk_r_1_2,
+    ofdm_bpsk_r_3_4,
+    ofdm_qpsk_r_1_2,
+    ofdm_qpsk_r_3_4,
+    ofdm_qam16_r_1_2,
+    ofdm_qam16_r_3_4,
+    ofdm_qam64_r_2_3,
+    ofdm_qam64_r_3_4
 };
 
 /** @brief Available bandwidths */
-enum Bandwidth {
-    BW_OFDM_5_MHZ,
-    BW_OFDM_10_MHZ,
-    BW_OFDM_20_MHZ
+enum class Bandwidth {
+    ofdm_5_mhz,
+    ofdm_10_mhz,
+    ofdm_20_mhz
 };
 
 /** @brief Given bandwidth and MCS returns datarate in bits per second */
```

Comment:
```suggestion
 /** Given bandwidth and MCS returns datarate in bits per second */
```

Reply:
I won't change this here since the documentation of `ConstsPhy.h` will become inconsistent otherwise.